ofi-cray/cray-tests
# Move rdm_* benchmarks from fabtests to cray-tests

These commits moves the rdm_\* benchmarks from fabtests including history.

It also gets the rdm benchmarks building in cray-tests:
- Move common code to include/ and common/
- Remove use of fabtests utilities
- Rename ft_\* to ct_*
- Rename PMI utils with prefix 'ctpm'
- Prefix tbarrier utils with 'ct'
- Remove wrapper scripts
- Rename rdma_one_sided to rdm_one_sided
- Remove README (no longer accurate or relevant)
- Remove incorrectly preserved copyrights

@hppritcha @e-harvey @jswaro 

Signed-off-by: Sung-Eun Choi sungeunchoi@users.noreply.github.com
